tests/test_serializers.py

```diff
@@ -0,0 +1,48 @@
+import pytest
```

Comment:
I think we should add test coverage for `JSONSerializer` here.

And also msgpack (though I assume that already has some coverage, doesn't hurt to add more)